denysyefimov_task1/denysyefimov_houseprices.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+from sklearn.metrics import mean_squared_error
+from sklearn import linear_model
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.model_selection import train_test_split
+
+
+def one_hot_encode(x):
+    return 1 if x == 'Partial' else 0
+
+
+def encode1(x):
+ if (x == 4):
+        return 3
+ else:
+        return x
+
+
+if __name__ == '__main__':
+    # add data in file
+    train = pd.read_csv('C:/Users/Gyrocopter_UA/PycharmProjects/pythonProject/data/train.csv')
+    test = pd.read_csv('C:/Users/Gyrocopter_UA/PycharmProjects/pythonProject/data/test.csv')
+
+    # show all data
+    plt.hist(train.SalePrice, color='black')
+    #plt.show()
+
+    # making some changes
+    deviation = np.log(train.SalePrice)
+    print("Deviation: ", deviation)
+    plt.hist(deviation, color='black')
+    #plt.show()

```

Comment:
Consider removing commented/unused code
And try to separate drawing from main logic - currently there are a lot of calls to `plt` that rendered but newer shown